outbound/adapter/dummy-outbound-filterer.ts

```diff
@@ -0,0 +1,6 @@
+import { identity } from "fp-ts/lib/function";
+import { OutboundFilterer } from "../port/outbound-filterer";
+
+export const create = <I>(): OutboundFilterer<I> => ({
```

Comment:
'dummy' seems something using for testing. 
Could we rename it with something like 'allow-all' ?